app/styles/govflanders/components/_modal.scss

```diff
@@ -97,3 +105,9 @@ $modal-content-padding: 1.5rem;
     }
   }
 }
+
+.vl-modal-dialog__content {
+  .data-table-pagination {
+    margin-bottom: 0em;
```

Comment:
0 values should be unitless. This is a new style rule, I will add it to Ygdir.

Reply:
I copied/refactored this code over from the main `app.scss`, but here the boyscout rule applies, so I will make the changes.